### ISSUE TITLE
Loss

### ISSUE THREAD

**Issue by zsgj-Xxx** [kbrodt/waymo-motion-prediction-2021#15]
Hi! 
I tried your method and I observed that in training, l2loss and log_softmax have so large difference. so my network does not learn multimodal tracks, only one best track is fitted. 
Do you have any solution?

**Comment by kbrodt** [comment]
Hi! We don't train with L_2 loss. We train with negative log-likelihood loss of Gaussian mixtures to capture the multimodality.

**Comment by zsgj-Xxx** [comment]
Oh. Sorry for my expression errors. What I use is the Loss in the code. However, my training results are inconsistent with the results in the paper. Among my 6 trajs, there is only one close to gt, others pred are near the original point
![Uploading mmexport1661165075704.png…]()

**Comment by kbrodt** [comment]
Do you observe this for all examples?

**Comment by zsgj-Xxx** [comment]
Yes, I observed nearly 2,000 samples in val, all got similar results, and they all got the best score in the same pred id.

**Comment by kbrodt** [comment]
It seems your model is overfitted. We didn't observe that behaviour. Do you train on full training dataset?

**Comment by zsgj-Xxx** [comment]
no yet, I just used 1/10 training data to verify the effect

**Comment by zsgj-Xxx** [comment]
Oh I found the problem.
I put the gt at the vehicle point instead of the origin in the preprocessing stage.
Thank you for your reply.

Best,
